src/styles/dotcms-theme/components/_tabview.scss

```diff
@@ -79,11 +79,5 @@
 }
 
 .p-tabview .p-tabview-nav .p-tabview-ink-bar {
-    z-index: 1;
-    display: block;
-    position: absolute;
-    bottom: 0;
-    height: $tabview-item-border-height;
-    background-color: $brand-secondary;
-    transition: 500ms cubic-bezier(0.35, 0, 0.25, 1);
+    display: none;
```

Comment:
Can't you just get rid of the class? or is taking props from some place else.

Reply:
no, because this element is visible in de `tabview` component